registration/tests/browser/browser_registration.py

```diff
@@ -82,8 +82,16 @@ def fill_form(self, form_data):
                 if bool(value) != element.is_selected():
                     element.click()
             elif not element.get_attribute('readonly'):
-                element.clear()
-                element.send_keys(value)
+                if not element.get_attribute('type') == 'hidden':
+                    element.clear()
+                    element.send_keys(value)
+                else:
+                    element_id = element.get_attribute('id')
+                    self.client.execute_script(
```

Comment:
@itsjeyd why is this necessary? Doesn't the controller code update the value of the hidden form field?